source/documentation/responsibility_model/responsibility_model.md

```diff
@@ -1,5 +1,11 @@
 # Know your responsibilities
 
+To be hosted on GOV.UK PaaS, your app must:
+
+- follow the [twelve-factor application principles](architecture.html#12-factor-application-principles); this will be true if your app has been written so it can deploy to another PaaS like Heroku
```

Comment:
```suggestion
- follow the [twelve-factor application principles](architecture.html#12-factor-application-principles); this will be true if you have written your app so it can deploy to another PaaS, such as Heroku
```